src/main/java/seedu/address/logic/commands/client/SortClientCommand.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.logic.commands.client;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.ClientCliSyntax.PREFIX_CLIENT_NAME;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_CLIENTS;
+
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.Prefix;
+import seedu.address.model.Model;
+import seedu.address.ui.Ui;
+
+/**
+ * Sort clients in address book.
+ */
+public class SortClientCommand extends ClientCommand {
+
+    public static final String COMMAND_FLAG = "-s";
+
+    public static final String MESSAGE_SUCCESS = "Sorted clients";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Sort clients in address book. "
+            + "Sort by name: "
+            + PREFIX_CLIENT_NAME + "0 (alphabetical) or "
+            + PREFIX_CLIENT_NAME + "1 (reverse alphabetical). "
```

Comment:
Just a general observation: 
Might be better to standardise as PREFIX_NAME for all, or have PREFIX_CLASS_NAME for all (CLASS here is our model classes).

Reply:
Standardised codebase to have `PREFIX_NAME`